src/game/ai.cpp

```diff
@@ -146,7 +146,7 @@ namespace ai
 
     int weappref(gameent *d)
     {
-        if(d->loadweap.length()) return d->loadweap[0];
+        if(d->loadweap.size()) return d->loadweap[0];
```

Comment:
I really appreciate the use of explicit comparisons. This implicitly casts the resulting numeric value to `bool`. This form is just cumbersome to read.

~~Also, isn't there an `empty()` function? It'd be a much more idiomatic approach to this. We should add one if there is none.~~ This was changed to an STL container, so there clearly is an `empty` function which you should use here.

Reply:
True that, solving